### PR TITLE
regenerate latest client gen to get custom object status scale api

### PR DESCRIPTION
**Pull request by juliantaylor**

### CODE REVIEW

File: kubernetes/client/apis/custom_objects_api.py [hunk 1]
```diff
@@ -672,6 +672,240 @@ def get_cluster_custom_object_with_http_info(self, group, version, plural, name,
...
+                                        files=local_var_files,
+                                        response_type='object',
+                                        auth_settings=auth_settings,
+                                        async_req=params.get('async_req'),
+                                        _return_http_data_only=params.get('_return_http_data_only'),
+                                        _preload_content=params.get('_preload_content', True),
+                                        _request_timeout=params.get('_request_timeout'),
+                                        collection_formats=collection_formats)
+
+    def get_cluster_custom_object_status(self, group, version, plural, name, **kwargs):
```
Comment (MahdiZareie) [inline kubernetes/client/apis/custom_objects_api.py]: I checked out to this branch and I tested this method.
this method returns 404 for an existing object,his sister (`get_namespaced_custom_object`)  with exactly the same parameters, returns the object successfully.

Comment (MahdiZareie) [reply kubernetes/client/apis/custom_objects_api.py]: I need this method :-S
https://github.com/kubernetes-client/python/issues/607

Comment (juliantaylor) [reply kubernetes/client/apis/custom_objects_api.py]: which version of kubernetes are you using? custom object subresources are enabled by default in kubernetes 1.11, in 1.10 you have to enable it with a feature gate.
You might also need to define the status in the custom resource definition, but I'm not sure about that, see https://github.com/kubernetes/sample-controller/blob/master/artifacts/examples/crd-status-subresource.yaml

**Comment by juliantaylor** [comment]
it appears a new client update was already done in master so this branch can be closed

**Pull request closed without merging**